functions/meetup.js

```diff
@@ -297,33 +296,85 @@
                 return null
               }
 
-              console.log(urlname, nextEvent)
               const meetup = processMeetupEvent(
                 await getEvent({
                   id: nextEvent,
                   urlname
                 })
               )
-              console.log(meetup)
               const ev = find(events, ['fields.linkToEvent.en-US', meetup.link])
               const entry = generateContentfulEvent({ ...meetup, ...group })
-              console.log(ev)
-              console.log(entry)
+              const presetHomepageFeatured = ev.fields.homepageFeatured
```

Comment:
Could I please have a sanity check that this will always remain the same rather even if ev is changed?